### DISCUSSION TITLE
GeoSearch passing mapOptions as prop

### DISCUSSION THREAD

**Discussion by nathan-schmidt-viget** [algolia/instantsearch#5349] [Q&A] [answered]
We would like to be able to pass `mapOptions` as a prop to `<GeoSearch>`  so that we add in options such mapID or styling to make the map colors all gray. 

I see that we have `mapOpitons` in the [component](https://github.com/algolia/react-instantsearch/blob/b850f25de3eb176ba93c531804219e7d2532cbf8/packages/react-instantsearch-dom-maps/src/GeoSearch.js#L36), so maybe there is a way to easily pass it in that I am not seeing or is not documented on https://www.algolia.com/doc/api-reference/widgets/geo-search/react/

Any direction or code snippets would appreciated.

**Comment by Haroenv** [comment, marked as answer]
Any other prop you pass to GeoSearch will get forwarded to the google maps creation: https://github.com/algolia/react-instantsearch/blob/b850f25de3eb176ba93c531804219e7d2532cbf8/packages/react-instantsearch-dom-maps/src/GoogleMaps.js#LL43

so something like `mapId: "8e0a97af9386fef"` would be `<GeoSearch mapId="8e0a97af9386fef" />`

**Reply by nathan-schmidt-viget** [reply]
That works perfectly!
